.htmlhintrc

```diff
@@ -0,0 +1,61 @@
+{
+  /*
+   * STANDARD
+   * =================
+   */
+  
+  // All tagnames must be in lowercase only
+  "tagname-lowercase": true,
+  
+  // All attribute names must be in lowercase
+  "attr-lowercase": true,
+  
+  // Attribute values must be enclosed by double quotemarks
+  "attr-value-double-quotes": true,
+  
+  // Attributes cannot be duplicated on a single tag
+  "attr-no-duplication": true,
+  
+  // Don't look for doctypes in the html files
+  "doctype-first": false,
+  
+  // Tags must be paired
+  "tag-pair": true,
+  
+  // Empty tags must be closed by self
+  "tag-self-close": true 
+  
+  // All special characters must be escaped
+  "spec-char-escape": true,
+  
+  // All ID's must be unique
+  "id-unique": true,
+  
+  // Values for all SRC's in <img>|<script>|<link> tags must be set
+  "src-not-empty": true,
+  
+  /*
+   * PERFORMANCE
+   * =================
+   */
+  
+  // Disallow script tags in <head>
+  "head-script-disabled": true,
```

Comment:
While I agree with this in principal, I think the argument can be made that by default it should be turned on unless the site knows they need it.

I would prefer that the devs get warned about a head script, and then they go turn off this feature if they absolutely must have the script in HEAD, instead of carelessly putting a script in HEAD which is generally agreed to be a bad practice. Often times the script is not necessary in HEAD but someone just put it there by mistake etc.

As for IE polyfills, I think the official browser support of IE9+ means that there will be few and far between polyfills.

What do you think? Does this approach or my thinking/reasoning behind it at least make more sense?